datacommons/places.py

```diff
@@ -92,3 +92,59 @@ def get_places_in(dcids, place_type):
   if isinstance(dcids, pd.Series):
     return pd.Series([result[dcid] for dcid in dcids])
   return result
+
+
+def get_pop_obs(dcid):
+  """ Returns :obj:`Place`'s population and observations.
+
+  Args:
+    dcids (:obj:`str`): Dcid of the place.
+
+  Returns:
+    An dictionary of population and observation related to this place.
+
+  Raises:
+    ValueError: If the payload returned by the Data Commons REST API is
+      malformed.
+
+  Examples:
+    We would like to get population and observation of Mountain View
+    `Mountain View <https://browser.datacommons.org/kg?dcid=geoId/0649670>`_.
+
+    >>> get_pop_obs("geoId/0649670")
+    {
+      'name': 'Mountain View',
+      'placeType': 'City',
+      'populations': {
+        'dc/p/013ldrstf6lnf': {
+          'numConstraints': 6,
+          'observations': [
+            {
+              'marginOfError': 119,
+              'measuredProp': 'count',
+              'measuredValue': 225,
+              'measurementMethod': 'CenusACS5yrSurvey',
```

Comment:
Can we choose BLS as the example population? We should try to avoid any observation that has measurementMethod "Cenus"ACS5YrSurvey until that's fixed in the graph.

Reply:
um, I made it Census now..